src/components/filters/PeopleFilters.tsx

```diff
@@ -0,0 +1,115 @@
+import { ChangeEvent } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../SearchLink';
+import { SexType } from '../../types';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
```

Comment:
If you dont have 'centuries' in url getAll will be a empty array, so || [] is useless

Reply:
Thank you, now I'll know, I didn't find this in the MDN
>The getAll() method of the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface returns all the values associated with a given search parameter as an array.

Done